authors/apps/articles/tests/article_tests_base_class.py

```diff
@@ -46,3 +46,9 @@ def setUp(self):
         self.non_existent_article = '/api/articles/hi/rate/'
         self.get_non_existent_article = '/api/articles/hi/ratings/'
         self.article_rating = '/api/articles/'
+        self.bookmark_article_url = '/api/articles/{}/bookmark/'.format(self.slug)
+
+
+
+
+
```

Comment:
@ElMonstro, why is there so many blank lines in this file?

Reply:
Will fix